Rocket.Chat/Models/Attachment.swift

```diff
@@ -61,7 +61,7 @@ class Attachment: BaseModel {
     var videoThumbPath: URL? {
         let path = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
         let documents = path[0]
-        return documents.appendingPathComponent("\(identifier ?? "temp").png")
```

Comment:
This is actually another bugfix.  Using this `identifier` to name the video thumb we were writing to disk but only being able to retrieve the cached thumb while the `Attachment` was in memory because every time we map an `Attachment` we set its identifier as a random string, so every time we load it on the chat we write a new file for the new random string identifier with the same data that should be cached. It was making a significant impact on our storage usage.

Reply:
Good finding! :-)